README.md

```diff
@@ -62,10 +62,10 @@ that the rule will create. It is comprised of a single predicate.
 Here is a rule that will infer an `uncle` relationship from existing data:
 
 ```
-  uncle(Nibling,Uncle) :- parent(Nibling,Parent), brother(Parent,Uncle).
+  uncle(Sibling,Uncle) :- parent(Sibling,Parent), brother(Parent,Uncle).
```

Comment:
"Nibling" is the correct term here. It is a gender-neutral form for niece or nephew.